pkgs/development/libraries/libnfc/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
i solved it with let, in. Which is hopefully also okay.